src/App.js

```diff
@@ -1,16 +1,45 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [likedMessages, setLikedMessages] = useState(0);
+  const [participants, setParticipants] = useState([]);
```

Comment:
Nice setup for your `likedMessages` and `participants` state variables at the very top of your component so it's easy to find!